todocore/src/main/java/com/knighttodo/todocore/character/response/ExpirienceResponse.java

```diff
@@ -0,0 +1,4 @@
+package com.knighttodo.todocore.character.response;
+
+public class ExpirienceResponse {
+}
```

Comment:
I suspect that I removed this class because we don't use it anywhere else, could you please check it and remove it if it's true.

Reply:
You exactly removed this class. But some tests and factory methods have links for this class and don't work And I come back to him. Besides the tests this class never used. I am removing him.